_posts/molecules/2018-11-26-collapse.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  Collapse
+category: molecules
+description: Toggle the visibility of content with a clickable icon
+---
+
+%a.request{ href: "/request/show/1" }
```

Comment:
`request{ href: "/request/show/1" }` is not really related to collapse, so I would remove it to remove complexity (same everywhere)